### DISCUSSION TITLE
Custom Flags?

### DISCUSSION THREAD

**Discussion by apikee** [pocketbase/pocketbase#1900] [Q&A] [answered]
Is it possible to allow custom flags when running PocketBase? Something like ./executable serve --mytag=verynice
Thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, you can add as many flags as you want by registering them to the `app.RootCmd` cobra command. As a reference You can check the prebuilt executable main file - https://github.com/pocketbase/pocketbase/blob/master/examples/base/main.go.

```go
var mytag string
app.RootCmd.PersistentFlags().StringVar(
    &mytag,
    "mytag",
    "default_value",
    "Lorem ipsum description...",
)
```

If you want to eagerly parse the flags before the `serve` or your custom command has started, you can run: `app.RootCmd.ParseFlags(os.Args[1:])`

**Reply by apikee** [reply]
great, thanks!
